server/render.js

```diff
@@ -80,7 +82,8 @@ function getTemplates() {
     return {
         BEMTREE: langs.reduce(function(acc, lang) {
             acc[lang] = evalFile(path.join(pathToBundle, bundleName + '.' + lang + '.bemtree.js')).BEMTREE;
-            return acc;
+
+return acc;
```

Comment:
tab

Reply:
what actually happened with tabs before return?)

Reply:
This was autofix in conjunction with disabled line https://github.com/bem/bem-forum/blob/master/.eslintrc.js#L88